### ISSUE TITLE
Compilation failed in React Native 0.48.3

### ISSUE THREAD

**Issue by ivan-magda** [mohtada-h/react-native-nested-scrollview#4]
Hey, @mohtada-h !

#### In React Native `0.48.3` I got some errors:

- [ReactNestedScrollViewHelper]: `ScrollEvent.obtain` method signature has changed. Added 2 additional parameters (`xVelocity`, `yVelocity`)
- [ReactNestedScrollViewManager]: `ScrollEventType` does not has `ANIMATION_END`
- [ReactNestedScrollViewPackage]: `createJSModules` - does not override method from its superclass

## ScrollEvent

`ScrollEvent`'s `obtaib` now has the following signature:

```java
public static ScrollEvent obtain(
      int viewTag,
      ScrollEventType scrollEventType,
      int scrollX,
      int scrollY,
      float xVelocity,
      float yVelocity,
      int contentWidth,
      int contentHeight,
      int scrollViewWidth,
      int scrollViewHeight) {
```

## ScrollEventType

`ScrollEventType` now it looks like this:
```java
public enum ScrollEventType {
  BEGIN_DRAG("topScrollBeginDrag"),
  END_DRAG("topScrollEndDrag"),
  SCROLL("topScroll"),
  MOMENTUM_BEGIN("topMomentumScrollBegin"),
  MOMENTUM_END("topMomentumScrollEnd");
...
}
```

```java
public static Map createExportedCustomDirectEventTypeConstants() {
    return MapBuilder.builder()
        .put(ScrollEventType.SCROLL.getJSEventName(), MapBuilder.of("registrationName", "onScroll"))
        .put(ScrollEventType.BEGIN_DRAG.getJSEventName(), MapBuilder.of("registrationName", "onScrollBeginDrag"))
        .put(ScrollEventType.END_DRAG.getJSEventName(), MapBuilder.of("registrationName", "onScrollEndDrag"))
        .put(ScrollEventType.MOMENTUM_BEGIN.getJSEventName(), MapBuilder.of("registrationName", "onMomentumScrollBegin"))
        .put(ScrollEventType.MOMENTUM_END.getJSEventName(), MapBuilder.of("registrationName", "onMomentumScrollEnd"))
        .build();
  }
```

## ReactPackage

`ReactPackage` interface now has only 2 methods:

```java
public interface ReactPackage {

  /**
   * @param reactContext react application context that can be used to create modules
   * @return list of native modules to register with the newly created catalyst instance
   */
  List<NativeModule> createNativeModules(ReactApplicationContext reactContext);

  /**
   * @return a list of view managers that should be registered with {@link UIManagerModule}
   */
  List<ViewManager> createViewManagers(ReactApplicationContext reactContext);
}
```

I was able to run the project after deleting `ScrollEventType.ANIMATION_END`, `createJSModules` and passing random values to velocity parameters.

Please take a look at this problem and do appropriate updates.

Thanks!

**Comment by smallpath** [comment]
It seems that this repo is not maintained any more. There is a pull request I sended [here](https://github.com/mohtada-h/react-native-nested-scrollview/pull/2) but gains nothing.

**Comment by ivan-magda** [comment]
@smallpath it’s sadly.

**Comment by Nyl000** [comment]
Fixed for 0.47.2 here https://github.com/mohtada-h/react-native-nested-scrollview/pull/5
Maybe works with your 0.48.3 too ?
